doc/source/gotchas.rst

```diff
@@ -242,6 +242,57 @@ Label-based slicing conventions
 Non-monotonic indexes require exact matches
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+If the index of a ``DataFrame`` is montonically increasing or decreasing, then the bounds
+of a label-based slice can be outside the range of the index, much like slice indexing a 
+normal Python ``list``.
+
+.. code-block:: python
+
+   >>> df_monotonic_index = pd.DataFrame(index=[2,3,3,4,5], columns=['a', 'b'])
+    
+    # no rows 0 or 1, but still returns rows 2, 3 (both of them), and 4:
+    >>> df_monotonic_index.loc[0:4, :]
+         a    b
+    2  NaN  NaN
+    3  NaN  NaN
+    3  NaN  NaN
+    4  NaN  NaN         
+    
+    >>> df_monotonic_index.loc[13:15, :]
+    Empty DataFrame
+    Columns: [a, b]
+    Index: []
+
+On the other hand, if the index is not monotonic, then both slice bounds must be
+*unique* members of the index. 
+
+.. code-block:: python
+
+    >>> df_nonmonotonic_index = pd.DataFrame(index=[2,3,4,5,3], columns=['a', 'b'])
+    >>> df_nonmonotonic_index.loc[2:4, :]
+         a    b
+    2  NaN  NaN
+    3  NaN  NaN
+    4  NaN  NaN                
+
+    # OK
+    >>> df_nonmonotonic_index.loc[2:4, :]
+         a    b
+    2  NaN  NaN
+    3  NaN  NaN
+    4  NaN  NaN    
+    
+    # 0 is not in the index
+    >>> df_nonmonotonic_index.loc[0:4, :]
+    Traceback (most recent call last):
+    ...
+    KeyError: 0
+    
+    >>> df_nonmonotonic_index.loc[2:3, :]
```

Comment:
maybe add here a comment like for the others "3 is not a unique label in the index"